34.md

```diff
@@ -181,6 +182,49 @@ The Status event with the largest created_at date is valid.
 
 The Status of a patch-revision defaults to either that of the root-patch, or `1632` (Closed) if the root-patch's Status is `1631` and the patch-revision isn't tagged in the `1631` event.
 
+## Git client nostr plugin (git-remote-nostr)
+
+Definition: Clone URL - a git remote URL listed under the `clone` tag of the repositories kind `30617` event
+
+The offical git client supports interacting with 'remotes' using custom protocols via [git-remote-helpers](https://git-scm.com/docs/gitremote-helpers).
+
+An implementation of git-remote-nostr SHOULD:
+
+### 1. Use Git Remote Nostr URL Format
+
+Struture:
+
+- `nostr://` - git syntax for a protocol URL
+- `<user>@` - optional ssh user for interacting with Clone URLs
+- `<protocol>/` - optional prefered protocol for interacting with Clone URLs. eg. ssh or https
+- kind `30617` event coordinate in format:
+  - `<naddr>`
+  - `<npub|nip05-address>/<identifier>`
+  - `<npub|nip05-address>/<relay-hint*>/<identifier>`
+
+\*relay hints SHOULD be [URL encoded](https://www.w3.org/Addressing/URL/4_Recommentations.html) eg `ws%3A%2F%2Funencrypted.com` and may omit `wss://` for brevity and readability.
+
+for example `nostr://dan@gitworkshop.dev/ngit` or `nostr://npub15qydau2hjma6ngxkl2cyar74wzyjshvl65za5k5rl69264ar2exs5cyejr/relay.damus.io/ngit`
+
+### 2. Manage state on nostr and access objects via Clone URLs
+
+Store and retrieve state in the `30618` event and git objects in the Clone URLs. Proxy state updates to each Clone URLs.
+
+If a `30618` cannot be found or git config item `nostr.nostate` is `true`, use state from the first Clone URL.
+
+### 3. Use `pr/*` branch namespace for 'Open' patches
```

Comment:
I would mention how to browse the draft PRs.

Reply:
this remote helper spec doesn't support it. The guidance here is to use other tools eg ngit. I'm not sure the mention of a specific client like ngit is appropriate for inclusion in a NIP.